xsnap/makefiles/mac/xsnap-worker.mk

```diff
@@ -122,13 +123,40 @@ OBJECTS = \
 	$(TMP_DIR)/xsdtoa.o \
 	$(TMP_DIR)/xsre.o \
 	$(TMP_DIR)/xsmc.o \
+ 	$(TMP_DIR)/e_acos.o \
+ 	$(TMP_DIR)/e_acosh.o \
+ 	$(TMP_DIR)/e_asin.o \
+ 	$(TMP_DIR)/e_atan2.o \
+ 	$(TMP_DIR)/e_atanh.o \
+ 	$(TMP_DIR)/e_cosh.o \
+ 	$(TMP_DIR)/e_exp.o \
+ 	$(TMP_DIR)/e_hypot.o \
+ 	$(TMP_DIR)/e_log.o \
+ 	$(TMP_DIR)/e_log10.o \
+ 	$(TMP_DIR)/e_pow.o \
+ 	$(TMP_DIR)/e_rem_pio2.o \
+ 	$(TMP_DIR)/e_sinh.o \
+ 	$(TMP_DIR)/k_cos.o \
+ 	$(TMP_DIR)/k_exp.o \
+ 	$(TMP_DIR)/k_rem_pio2.o \
+ 	$(TMP_DIR)/k_sin.o \
+ 	$(TMP_DIR)/k_tan.o \
+ 	$(TMP_DIR)/s_asinh.o \
+ 	$(TMP_DIR)/s_atan.o \
+ 	$(TMP_DIR)/s_cos.o \
+ 	$(TMP_DIR)/s_expm1.o \
+ 	$(TMP_DIR)/s_log1p.o \
+ 	$(TMP_DIR)/s_scalbn.o \
+ 	$(TMP_DIR)/s_sin.o \
+ 	$(TMP_DIR)/s_tan.o \
+ 	$(TMP_DIR)/s_tanh.o \
 	$(TMP_DIR)/textdecoder.o \
 	$(TMP_DIR)/textencoder.o \
 	$(TMP_DIR)/modBase64.o \
 	$(TMP_DIR)/xsnapPlatform.o \
 	$(TMP_DIR)/xsnap-worker.o
 
-VPATH += $(SRC_DIR) $(TLS_DIR)
+VPATH += $(SRC_DIR) $(TLS_DIR) $(XS_TLS_DIR)/fdlibm
```

Comment:
This is only slightly different than what I found in `xst.mk`. The difference accounts for `xsnap` tools being a different than `xs` tools.